gnovm/pkg/gnomod/pkg.go

```diff
@@ -5,14 +5,18 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"slices"
 	"strings"
+
+	"github.com/gnolang/gno/gnovm/pkg/gnoimports"
+	"github.com/gnolang/gno/gnovm/pkg/gnolang"
 )
 
 type Pkg struct {
-	Dir      string   // absolute path to package dir
-	Name     string   // package name
-	Requires []string // dependencies
-	Draft    bool     // whether the package is a draft
+	Dir     string // absolute path to package dir
+	Name    string // package name
+	Imports []string
```

Comment:
There should be a comment on the fact that this is transitional as well

Reply:
No, it's only direct

Reply:
Yeah, but we're using this for module imports while we should treat imports as something related to _packages_ rather than _modules_. it goes in hand with what I was saying about having a `packages` package; we should have package analysis and `imports` stuff there.

Reply:
Fully refactoring packages loading is out of scope of this PR, this is focused on replacing `require` with imports analysis
I don't understand, what do you want me to do there?

Reply:
I'll add a comment here directly, and make a fuller comment in the review

Reply:
Actually, it's fine ish. I still think there's some renaming / re-organizing to be done, but i'll close this up for now.